services/auth.js

```diff
@@ -42,11 +41,13 @@ const authService = {
       firstName,
       lastName,
       email,
-      password: hashedPassword,
-      activationLink
+      password: hashedPassword
     })
 
-    await sendEmail(email, emailSubject.EMAIL_CONFIRMATION, { activationLink, email, firstName })
+    const confirmToken = tokenService.generateConfirmToken({ id: user._id, isEmailConfirmed: false })
+    await tokenService.saveToken(user._id, confirmToken, CONFIRM_TOKEN)
+
+    await sendEmail(email, emailSubject.EMAIL_CONFIRMATION, { confirmToken, email, firstName })
```

Comment:
please, discuss with your teammates case when something went wrong with email sending

Reply:
thank you for the suggestion, I shared it, will discuss (: